src/core/ProductTile/data.ts

```diff
@@ -64,6 +64,6 @@ export const products: Products = {
     description: "Sync application state across client devices.",
     icon: "icon-product-liveobjects-mono",
     hoverIcon: "icon-product-liveobjects",
-    link: "/docs/liveobjects",
+    link: "/liveobjects",
```

Comment:
@kennethkalmer  this change will affect the Pubsub page - https://ably.com/pubsub which they are link to docs.
to update the link for meganav it is here - https://github.com/ably/ably-ui/blob/ec05c93daf59c854a899d9b2c2e09dedd98f8ced/src/core/Meganav/data.tsx#L210

Reply:
@aralovelace that object pulls in `...products`, which comes from the `src/core/ProductTile/data.ts`. I can't find any other references in the project to `/liveobjects' o_O

Reply:
@aralovelace I've reworked this based on your feedback in Slack :)

Reply:
From what I can remember making this they all link to docs pages, so this probably should as well.

We should add a way to differentiate on links depending on context, but that's a tomorrow job.